Tavisca.Bootcamp.LanguageBasics.Exercise1/Program.cs

```diff
@@ -14,36 +14,21 @@ static void Main(string[] args)
             Console.ReadKey(true);
         }
 
-        private static void Test(string args, int expected)
-        {
+        private static void Test(string args, int expected)
+        {
             var result = FindDigit(args).Equals(expected) ? "PASS" : "FAIL";
-            Console.WriteLine($"{args} : {result}");
+            Console.WriteLine($"{args} : {result}");
         }
 
-        public static int FindDigit(string equation)
-        {
+        public static int FindDigit(string equation)
+        {
             String[] expressions = equation.Split(new Char[] { '*', '=' });        //here we split the equation into three parts
             int[] initialize = new int[3];                                         //here we made an array for basically storing at which part is an '?'

```

Comment:
If this is used to store the location of ?, why not name it questionMarkLocation?
Aim is to name the variables so that they tell what they are used for, which makes it easy to read the code.